R/data.R

```diff
@@ -1,4 +1,4 @@
-#' Example wastewater dataset.
+#' Example wastewater dataset with site correlations from correlation func.
```

Comment:
Maybe specify from an exponential correlation function with the specified distance matrix